app/webpacker/stylesheets/settings/_colors.scss

```diff
@@ -52,7 +52,7 @@ $i-belong-cert-color: $orange;
   background-color: $deep-cerulean;
 }
 .dixie-bg {
-  background-color: $dixie;
+  background-color: $dixie !important;
```

Comment:
Could you make a separate class for the important style? That way it won't affect other parts of the code which may want to override it without using !important

Reply:
resolved thank you

Reply:
@nyashamaphosa do you need to push your branch, I don't see this change?